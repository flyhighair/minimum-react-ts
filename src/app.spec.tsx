import React from 'react';
import { render } from '@testing-library/react';
import App from './app';

test('Check Title', () => {
  const component = render(<App />);
  expect(component.getByRole('heading').innerHTML).toContain('Hello World');
});
