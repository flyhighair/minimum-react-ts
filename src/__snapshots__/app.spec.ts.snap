// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<h1>
  Hello World
</h1>
`;
